keyboards/creatkeebs/thera/readme.md

```diff
@@ -0,0 +1,22 @@
+# thera
+A 75% keyboard
+
+![thera](https://i.imgur.com/7w7ARcP.jpg)
+
+### Features
+* Plateless PCB gasket
+* Decent amount of layout compatibility - [View supported layouts](https://cdn.shopify.com/s/files/1/0481/5840/8853/products/8038a8b71a5149f5bad362b28368da3_590x.png?v=1628765539)
+* Ti badge
+* PVD weight (upgrade available)
+
+### Additional Information
+* Keyboard Maintainer: [Tim](https://github.com/Timliuzhaolu/qmk_firmware)
+* Hardware Supported: thera PCB
+* Hardware Availability: [ESCHIT.com](https://eschit.com/products/gb-thera75)
+
+### Building the Firmware
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```